unfoldingWord/translationCore
# Fix USFM Project Overwrite Dialog

#### This pull request addresses:
This PR simply changes some text in a dialog when you try to import the same usfm project twice.


#### How to test this pull request:
Try importing the same usfm project twice and ensure that it shows the dialog.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2408)
